readme.md

```diff
@@ -80,30 +84,38 @@ We're happy to discuss design patterns and learn how you're using Marionette.
 
 <a href="https://github.com/marionettejs/marionette.inspector"><img src="http://i.imgur.com/B1q9QXH.jpg" align="center" /></a>
```

Comment:
so the inspector is kind of broken right now and isn't going to work with v3.. maybe we should pull it for now?